tests/src/test/scala/services/KafkaConnectorTests.scala

```diff
@@ -65,17 +65,34 @@ class KafkaConnectorTests extends FlatSpec with Matchers with WskActorSystem wit
     super.afterAll()
   }
 
+  def commandComponent(host: String, command: String, component: String) = {
```

Comment:
@devbv Could reuse these methods here: 
https://github.com/apache/incubator-openwhisk/blob/e587bf8ec632320614f14114c4e85e67f39c06c2/tests/src/test/scala/ha/ShootComponentsTests.scala#L94

Sorry I'm a little late to the party.

Reply:
@rabbah ok, I could fix it on another PR including enhancement of handling kafka errors.